### ISSUE TITLE
Griffon Hibernate 4 plugin doesn't set all values from "Hibernate4.groovy

### ISSUE THREAD

**Issue by swalton00** [griffon/griffon#46]
Severa properties in the Hibernate4.groovy configuration file are not passed in to Hibernate (most are however). Generate_statistics and default_schema appear to only be effective if set in the "hibernate.properties" file. The "hibernate.cfg.xml" is apparently ignored. Because the hibernate.properties applies to all environments (dev, test and prod), it is not possible to set these properties on a per-environment basis. If you set the default schema with the database URL, hibernate is not aware of that and does not correctly verify (it verifies the schema of the logged in user,not the default).

Connect_on_startup doesn't actually connect until the the first reference to a hibernate entity. If you place the first reference in a secondary window, edit the hibernate.properties to contain illegal values (or set it up so the mapping is incorrect), the error is not caught until the secondary window is opened and hibernate is configured.

**Comment by swalton00** [comment]
I did some testing over the last few days. In every case I've found it is possible to use a database specific technique to set the schema and Hibernate works correctly. Just two points then:
1) default_schema in the hibernate settings (for each environment, dev, test, prod) is much better approach then modifying the URL in the Datasource.groovy with a DBMS specific setting. and
2)settings the Hibernate4.groovy that are not valid should at least result in a warning message, rather than being silently ignored.

Given my testing I'm okay with this being treated as an "enhancement request" rather than a bug.
